static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js

```diff
@@ -4,15 +4,18 @@ import config from 'lib/config';
 const insertSnippet = () => {
     // Twitter universal website tag code
     // How to set up conversion tracking: https://business.twitter.com/en/help/campaign-measurement-and-analytics/conversion-tracking-for-websites.html
-    /*eslint-disable */
+    /* eslint-disable */ // prettier-ignore
     // $FlowFixMe
-        !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe(...arguments):s.queue.push(arguments);
+    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe(...arguments):s.queue.push(arguments);
     },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
         a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
-        // Insert Twitter Pixel ID and Standard Event data below
-        // $FlowFixMe
-        twq('init','nyl43');
-        twq('track','PageView');
+    // Insert Twitter Pixel ID and Standard Event data below
+    // $FlowFixMe
+    twq('init', 'nyl43'); // The Guardian ID
+    twq('track', 'PageView');
+
+    twq('init', 'ny4k9'); // PHD ID
```

Comment:
nice generalisation if it works inside the same script !
Does it show 2 scripts when running the [twitter pixel helper plugin](https://chrome.google.com/webstore/detail/twitter-pixel-helper/jepminnlebllinfmkhfbkpckogoiefpd)?

Reply:
![both twitter tracker working](https://user-images.githubusercontent.com/76776/94812256-66feb380-03ee-11eb-9984-99f31087abda.png)